lib/tic_tac_toe_checker/application.ex

```diff
@@ -26,11 +27,13 @@ defmodule TicTacToeChecker.Application do
   end
 
   @impl Application
+  @spec prep_stop(any) :: :ok
   def prep_stop(state) do
-    IO.inspect(state)
+    IO.puts(state)
```

Comment:
I doubt this would work for `state :: any()`. `String.Chars` has no `any` fallback.

Reply:
Changed to `Logger`.